src/UX/Collector/MainMenu/Main.php

```diff
@@ -0,0 +1,40 @@
+<?php namespace ILIAS\UX\Collector;
+
+use ILIAS\UX\MainMenu\Slate\Slate;
+
+/**
+ * Class Main
```

Comment:
Add a sentence, what this class does. I am really not sure atm. Is this not the class that is supposed to be in `/Services/StaticMainMenuProvider` (see readme). If not, what is the relation among those to classes.

Reply:
Done :-)